llama/runner/runner.go

```diff
@@ -876,6 +878,31 @@ func (s *Server) loadModel(
 		}
 	}
 
+	if cvpath.String() != "" {
+		// format path=strength
```

Comment:
To pass the strength to the runner, I decided to use `-control-vector /some/path=0.5` syntax rather complicating it by adding two disjoint flags